sections/oxygen_sensors_integration.md

```diff
@@ -62,21 +72,21 @@ After this it should be less than 0.5 % per year, unless the foil is mechanicall
 :::{figure-md} optodes
 <img src="/images/AaanderaasensorsAll1.png" alt="Aandera Optodes" class="bg-primary mb-1" width="400px">
 
-Suit of smart optodes sensors. Oxygen sensors are indicated by red arrows.
+Suit of Aanderaa smart sensors. Oxygen optodes are indicated by red arrows.
 :::
 
 ### RBR coda T.ODO
 The RBRcoda T.ODO uses the same foils and methods as the 4831 and 4831F so everything above specified for the 4831 will also apply to those instruments as well.
 RBR refers to the standard optode (~30 s τ) foil as “slow” and the fast (~8 s) as the standard.
-They also use further foil design (~1 s response) which they call fast. 
-The RBR sensor has a smaller form factor than the Aandera optodes, but is overall more similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
+They also use further foil design (~1 s response) which they call “fast” . 
+The RBR sensor has a smaller form factor than the Aandera optodes, but is overall similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
 This sensor has recently been implemented in gliders, and little is known about their performance.
 
 ### JFE Advantech RINKO
 AROD-FT sensor (RINKO JFE) is used for the SeaExplorer gliders (Alseamar) and for some Argo floats (small size and low power consumption) (see {numref}`ARODFT`). 
-This sensor is based on the optical (phosphorescence) principle which is now widely known as a remarkably fast response oxygen sensor (below 1s) with a high accuracy of 2 $\mu$mol/kg. 
+This sensor is based on the optical (phosphorescence) principle which a remarkably fast response oxygen sensor (below 1 s) with a good accuracy (±2 μmol kg<sup>-1</sup>). 
 This sensor used a multi-points calibration (16 points with 4 temperatures and 4 DO concentrations). 
-In this procedure, the DO reference standards are produced by saturing the primary mixtures with DO concentrations of approximately 4%, 10%, 17% and 25% respectively (certified by the National Metrology Institute of Japan). 
+The DO reference standards are produced by saturating the primary mixtures with DO concentrations of approximately 4%, 10%, 17% and 25% respectively (certified by the National Metrology Institute of Japan). 
```

Comment:
Are these reference standards for the multi-point calibration from the preceding sentence ? Then this should be
'The DO reference standards used for the multi-point calibrations are...'